contributing_to_docs/doc_guidelines.adoc

```diff
@@ -692,6 +692,7 @@ In OpenShift docs:
 * All links to internal content is created using `xref` and **must have an anchor ID**.
 * Only use `xref` in assemblies, not in modules.
 * All links to external websites are created using `link`.
+* Links between different distros, such as from ROSA to OpenShift Container Platform, are external links. They use `link` instead of `xref` and need to be in distro-specific files or conditionalized to apply to the relevant distros.
```

Comment:
```suggestion
* Links between different distros, such as from ROSA to OpenShift Container Platform, are external links and not cross-references. These external links use `link` instead of `xref` and need to be in distro-specific files or conditionalized to apply to the relevant distros.
```